console/src/view/resources.rs

```diff
@@ -23,13 +23,16 @@ pub(crate) struct ResourcesTable {}
 impl TableList for ResourcesTable {
     type Row = Resource;
     type Sort = SortBy;
+    type Context = ();
 
     const HEADER: &'static [&'static str] = &[
         "ID",
+        "Parent",
         "Kind",
         "Total",
         "Target",
         "Type",
+        "Viz",
```

Comment:
i have a slight preference for spelling this "Vis", as it's short for "Visibility"...or maybe it should just be "pub"? but, i could be convinced otherwise...